lms/static/scripts/frontend_apps/components/InstructorToolbar.tsx

```diff
@@ -31,12 +31,24 @@ export default function InstructorToolbar() {
       )}
     >
       <div className="space-y-1">
-        <h1
-          className="text-lg font-semibold leading-none"
-          data-testid="assignment-name"
-        >
-          {assignmentName}
-        </h1>
+        <div className="flex gap-x-2 items-center">
+          <h1
+            className="text-lg font-semibold leading-none"
+            data-testid="assignment-name"
+          >
+            {assignmentName}
+          </h1>
+          {editingEnabled && (
+            <LinkButton
+              classes="text-xs"
```

Comment:
I know you're planning to explore alternatives here soon, but maybe change to `text-sm`. The `text-xs` size is only 12px. Preview with `text-sm`:

<img width="1188" alt="text-sm" src="https://user-images.githubusercontent.com/2458/220342464-f0391301-5d90-4bd6-9dd7-9511e2b1d656.png">

Reply:
I gently agree: the font size is too small. However, what I'm after is a size contrast between the assignment-name font and the button's text. The reasoning for this is because the button is center-aligned, vertically, here, and if the font size is too close to the heading font size, it looks awkward and floaty. I'm going to merge this as-is so that we have a non-broken interface here, but will continue to improve.